config/secrets.yml

```diff
@@ -36,5 +37,6 @@ production:
   secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
   api_name: <%= ENV["API_NAME"] %>
   api_password: <%= ENV["API_PASSWORD"] %>
+  github_password: <%= ENV["GITHUB_PASSWORD"] %>
```

Comment:
I have set my github password in bashrc file and using it as an environment variable here. This is the way to keep our passwords secret, right?